packages/grafana-ui/src/components/Logs/logParser.ts

```diff
@@ -99,3 +86,28 @@ function sortFieldsLinkFirst(fieldA: FieldDef, fieldB: FieldDef) {
   }
   return fieldA.key > fieldB.key ? 1 : fieldA.key < fieldB.key ? -1 : 0;
 }
+
+function shouldRemoveField(field: Field, index: number, row: LogRowModel) {
+  // Remove field if it is:
+  // "labels" field that is in Loki used to store all labels
+  if (field.name === 'labels' && field.type === FieldType.other) {
+    return true;
+  }
+  // "id" field which we use for react key
+  if (field.name === 'id') {
+    return true;
+  }
+  // entry field which we are showing as the log message
+  if (row.entryFieldIndex === index) {
+    return true;
+  }
+  // hidden field
+  if (field.config.custom?.hidden) {
+    return true;
+  }
+  // field that has empty value (we want to keep 0 or empty string)
+  if (field.values.get(row.rowIndex) == null) {
```

Comment:
```suggestion
  if (field.values.get(row.rowIndex) === null || field.values.get(row.rowIndex) === undefined) {
```

I'm surprised no linter complained about using strict equality (===). Do we not consider it a bad practice in Grafana?

Reply:
Gotcha, thanks for clarifying!